arrayMethod_Opdracht_1.js

```diff
@@ -0,0 +1,32 @@
+//Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+
+const allTvTypes = inventory.map((tvs) => {
+    return 'Tv Types: ' + tvs.type;
+});
+
+console.log(allTvTypes);
+
+//Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+
+const soldOut = inventory.filter((tvs) => {
+    return tvs.originalStock === tvs.sold;
+});
+
+console.log(soldOut)
+
+//Opdracht 1c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken. Log de uitkomst in de console.
+
+const ambiLight = inventory.find((tvs) => {
+    return tvs.options.ambiLight === true;
```

Comment:
`boolean_waarde === true` is altijd gelijk aan `boolean_waarde`, dus in dit geval
```suggestion
    return tvs.options.ambiLight;
```